components/Analytics.tsx

```diff
@@ -0,0 +1,43 @@
+import { usePathname, useSearchParams } from 'next/navigation';
+import Script from 'next/script';
+import { useEffect } from 'react';
+
+import { pageview } from '../utils/gtm';
+
+/**
+ * @description The Analytics is the for Google Analytics 4 with GTM.
+ */
+export default function Analytics() {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+
+  useEffect(() => {
+    if (window.location.hostname === 'asyncapi.com') {
```

Comment:
> am I correct seeing that with your change google tag script is always fetched, even if website started on localhost?
@derberg It should prevent it from running outside of asyncapi.com, when using `window.location.hostname.includes`, I got security warning that's why I changed this approach